selfdrive/updated/tests/test_updated.py

```diff
@@ -183,6 +186,37 @@ def test_switch_branches(self):
       self._test_params("master", False, True)
       self._test_update_params("master", *self.MOCK_RELEASES["master"])
 
+  def test_agnos_update(self):
+    # Start on release3, push an update with an agnos change
+    self.setup_remote_release("release3")
+    self.setup_basedir_release("release3")
+
+    with mock.patch("openpilot.system.hardware.AGNOS", "True"), \
+         mock.patch("openpilot.system.hardware.tici.hardware.Tici.get_os_version", "1.2"), \
+         mock.patch("openpilot.system.hardware.tici.agnos.get_target_slot_number"), \
+         mock.patch("openpilot.system.hardware.tici.agnos.flash_agnos_update"), \
```

Comment:
TODO: find a way to check that these have been called. Doesn't work with multiprocessing